scripts/runtime-weight-template.hbs

```diff
@@ -1,47 +0,0 @@
-//! Autogenerated weights for {{pallet}}
-//!
-//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION {{version}}
-//! DATE: {{date}}, STEPS: `{{cmd.steps}}`, REPEAT: {{cmd.repeat}}, LOW RANGE: `{{cmd.lowest_range_values}}`, HIGH RANGE: `{{cmd.highest_range_values}}`
-//! EXECUTION: {{cmd.execution}}, WASM-EXECUTION: {{cmd.wasm_execution}}, CHAIN: {{cmd.chain}}, DB CACHE: {{cmd.db_cache}}
```

Comment:
Why can we remove the bench template?

Reply:
We're not using it, if we do not put any template we're using the default. That it's how it's working right now.